src/main/java/com/redhat/sast/api/service/StatisticService.java

```diff
@@ -152,4 +153,31 @@ private OshScanStatusDto convertUncollectedScanToDto(OshUncollectedScan uncollec
         dto.setProcessedAt(uncollectedScan.getCreatedAt());
         return dto;
     }
+
+    /**
+     * Gets job activity statistics for the last 24 hours.
+     * Returns hourly data points showing job counts by status.
+     *
+     * @return list of 24 hourly data points with job counts
+     */
+    public List<JobActivityDataPointDto> getJobActivity24h() {
```

Comment:
I would not create a method strictly for 24h time period. Maybe you could think about a generic solution that can retrieve any number of hours if you'd like.